gradle/verification-metadata.xml

```diff
@@ -1761,6 +1772,11 @@
             <sha256 value="e969402c143f527967fdfec29ce756a75e29ea0f3104bf99ab3406dac03a6767" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.apache.commons" name="commons-lang3" version="3.11">
```

Comment:
We seem to now have a lot of commons-lang3 versions. Do you know which ones are still used and which ones aren't? I suggest we select one of the many.

Reply:
I've locked this dependency to the latest release (3.12.0) via the distribution platform and removed all other entries in the verification data file in 6dbfe7791dce6ccb4f15bf04664f8167a8666bb6